doc/api/http.md

```diff
@@ -2061,8 +2061,13 @@ order:
     (`'data'` will not be emitted at all if the response body is empty, for
     instance, in most redirects)
   * `'end'` on the `res` object
+* `'error'` if the server response is received while still writing to request.
```

Comment:
I'm not sure that this belongs here as it's a list of events for a *successful* request.

Reply:
depends on what you define as a successful request, anything that has a `response` is successful?

Reply:
I would say any request and response that completes without any `'error'` events.